benchmarks/TurboMqtt.Benchmarks/Mqtt311/Mqtt311End2EndTcpBenchmarks.cs

```diff
@@ -83,7 +83,8 @@ public void StartFixture()
             Password = "testpassword",
             KeepAliveSeconds = 60,
             MaxReconnectAttempts = 10,
-            PublishRetryInterval = TimeSpan.FromSeconds(5)
+            PublishRetryInterval = TimeSpan.FromSeconds(5), 
+            MaximumPacketSize = 16 * 1024
```

Comment:
Maximum package size is 8K, causing 8K benchmark to fail

Reply:
We should probably just make the global default bigger, like 32kb (for big JSON payloads that get encoded into UTF8)